test/draw/gpu/ToRasterTest.ooc

```diff
@@ -18,18 +18,35 @@ ToRasterTest: class extends Fixture {
 		gpuImage := context createImage(sourceImage)
 		raster := gpuImage toRaster()
 		expect(raster distance(sourceImage), is equal to(0.0f))
-		sourceImage free()
+		gpuImage free()
+		raster free()
+	}
+	toRasterTargetTestFunction: func (sourceImage: RasterImage) {
+		gpuImage := context createImage(sourceImage)
+		raster := sourceImage create(sourceImage size) as RasterImage
+		gpuImage toRaster(raster)
+		expect(raster distance(sourceImage), is equal to(0.0f))
 		gpuImage free()
 		raster free()
 	}
 	init: func {
 		super("ToRaster")
-		this add("toRaster bgra", || this toRasterTestFunction(RasterBgra open("test/draw/gpu/input/Flower.png")))
-		this add("toRaster monochrome", || this toRasterTestFunction(RasterMonochrome open("test/draw/gpu/input/Flower.png")))
-		this add("toRaster yuv", || this toRasterTestFunction(RasterYuv420Semiplanar open("test/draw/gpu/input/Flower.png")))
+		this add("toRaster bgra", || this toRasterTestFunction(bgra))
+		this add("toRaster monochrome", || this toRasterTestFunction(monochrome))
+		this add("toRaster yuv", || this toRasterTestFunction(yuv))
+
+		this add("toRasterTarget bgra", || this toRasterTargetTestFunction(bgra))
+		this add("toRasterTarget monochrome", || this toRasterTargetTestFunction(monochrome))
+		this add("toRasterTarget yuv", || this toRasterTargetTestFunction(yuv))
 	}
 }
 
+bgra := RasterBgra open("test/draw/gpu/input/Flower.png")
+monochrome := RasterMonochrome open("test/draw/gpu/input/Flower.png")
+yuv := RasterYuv420Semiplanar open("test/draw/gpu/input/Flower.png")
```

Comment:
This should really be in the `init` function of the test, which should also include a `free` method which calls the `free` methods on line 50-52. If you're changing the thing above, do this too, otherwise don't bother, it's a test.